GoogleSignIn/Sources/GIDProfileData_Private.h

```diff
@@ -28,6 +36,9 @@ NS_ASSUME_NONNULL_BEGIN
                    familyName:(nullable NSString *)familyName
                      imageURL:(nullable NSURL *)imageURL;
 
+/// Initialize with id token.
+- (instancetype)initWithIDToken:(OIDIDToken *)idToken;
```

Comment:
Mark the return nullable as the implementation could return nil.

Reply:
Good catch.